.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,28 @@
+{
+    "name": "Plain JF Dotnet 8",
```

Comment:
Why this name, instead of just "Jellyfin" or similar? We always develop against the latest .NET version, so is there any reason to delineate?

Reply:
Well i plan to make multiple versions for different scenarios for example:
- Development Jellyfin Server 
- Development Jellyfin Server - JFFmpeg
- Development Jellyfin Server - Standard FFmpeg
- Development Jellyfin Server - JFFmpeg with Media files
- Test Jellyfin Server
- Test Jellyfin Server - JFFmpeg
- Test Jellyfin Server - Standard FFmpeg
- Test Jellyfin Server - JFFmpeg with Media files
- Test Jellyfin Server - Multi User
- Test Jellyfin Server - DLNA

And so on. The dotnet8 moniker was because when i started server still targeted and older net version and you cannot use it currently to run the 10.8.13 release for example.

Reply:
So this one should be `Development Jellyfin Server` right? We shouldn't need to worry about ever supporting multiple versions, since once 10.9 is released, even when we upgrade master to .NET 9, the version of this file on the 10.9.x branch would still target .NET 8, no?

Reply:
> So this one should be Development Jellyfin Server right? 

Yes
> even when we upgrade master to .NET 9, the version of this file on the 10.9.x branch would still target .NET 8, no?

That is actually a good point.